assets/app/game_manager.rb

```diff
@@ -125,8 +131,14 @@ def url(game, path = '')
     "/game/#{game['id']}#{path}"
   end
 
-  def hs_url(game)
-    "/hotseat/#{game['id']}"
+  def hs_url(game, game_data)
```

Comment:
how can hot seats ever get pinned?

Reply:
Pass ?pin_version=githash

This would be used if someone reported a broken hotseat to us, we can get them to set the relevant hash.